samples/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -233,6 +247,41 @@ static bool event_handler(const struct event_header *eh)
 		return false;
 	}
 
+	if (IS_ENABLED(CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE)) {
+		if (is_config_forward_event(eh)) {
+			struct config_forward_event *event = cast_config_forward_event(eh);
+
+			if (!bt_gatt_hids_c_ready_check(&hidc)) {
+				LOG_WRN("Cannot forward, peer disconnected");
```

Comment:
is other module waiting for config sent?

Reply:
The only thing that will happen here is that python tool will get info that config was not delivered; it will retry few times (currently 5 times in 500ms intervals) and give up.

Reply:
Can't we response in case of error? It seems better for me if we know transfer failed...